app/controllers/finance/npq/contracts_controller.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Finance
+  module NPQ
+    class ContractsController < BaseController
+      def index
+        @npq_lead_providers = LeadProvider.all
```

Comment:
is this `#index` used? if it is then i don't think `LeadProvider.all` is correct as this is the ECF ones